source/docs/yearly-overview/yearly-changelog.rst

```diff
@@ -165,17 +165,19 @@ OutlineViewer has been updated to be C++ based using the ImGui library. This mak
 WPILib All in One Installer
 ---------------------------
 
-- No new changes
+- Visual Studio Code has been updated to 1.62
+- Updated Java and C++ language extensions
+- Fix Linux desktop icon permissions
```

Comment:
Did this issue exist in 2021?

Reply:
yes, although it wasn't reported until the beta